src/main/kotlin/com/dansplugins/fiefs/command/fiefs/list/FiefsListCommand.kt

```diff
@@ -0,0 +1,31 @@
+package com.dansplugins.fiefs.command.fiefs.list
+
+import com.dansplugins.fiefs.Fiefs
+import org.bukkit.ChatColor
+import org.bukkit.command.Command
+import org.bukkit.command.CommandExecutor
+import org.bukkit.command.CommandSender
+import org.bukkit.command.TabCompleter
+
+class FiefsListCommand(private val plugin: Fiefs) : CommandExecutor, TabCompleter {
+    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<out String>): Boolean {
+        if (plugin.fiefRepository.getFiefs().isEmpty()) {
```

Comment:
Assuming the fief repository will read these from the database, you may wish to be careful about threading - commands are run on the main thread, and database queries perform I/O which would cause lag spikes each time the command was run.
Additionally, I'd recommend adding another layer of abstraction in a fief service type thing (which contains any domain logic) so your command isn't interfacing directly with the database.